src/Features/Core/Portable/Completion/CompletionTrigger.cs

```diff
@@ -28,22 +29,19 @@ internal CompletionTrigger(CompletionTriggerKind kind, char character = (char)0)
         /// <summary>
         /// The default <see cref="CompletionTrigger"/> when none is specified.
         /// </summary>
+        [Obsolete("Do not use.", error: true)]
```

Comment:
>[Obsolete("Do not use.", error: true)] [](start = 8, length = 38)

It seems bizarre to have a Default that should not be used.  Should the default just be changed to default to Invoke instead?